docs/libcurl/libcurl-multi.3

```diff
@@ -98,7 +98,12 @@ period for your select() calls.
 one of its input arguments, and by reading that you can figure out when all
 the transfers in the multi handles are done. 'done' does not mean
 successful. One or more of the transfers may have failed. Tracking when this
-number changes, you know when one or more transfers are done.
+number changes, you know when one or more transfers are done. Please note, that
+transfer may fail immediately, especially in case you are adding/re-using easy
+handles into already running multi handle. At the same time newly added
+transfer is not guaranteed to start immediately. In both cases
+\fIcurl_multi_perform(3)\fP will not count that handle as running transfer,
```

Comment:
This is incorrect. The number of "running handles" is increased by `curl_multi_add_handle` so a slow start doesn't result in a lower number. The number is however decreased when the transfer is complete, which can happen at any point and thus already in the fist call to `curl_multi_perform()`.

Reply:
Rephrased to be more precise.

Reply:
How about instead trying to not duplicate what the curl_multi_perform.3 man page already says ? We can instead rather remove the (possibly slightly misleading) sentence "Tracking when this number changes, you know when one or more transfers are done."

Reply:
Agree